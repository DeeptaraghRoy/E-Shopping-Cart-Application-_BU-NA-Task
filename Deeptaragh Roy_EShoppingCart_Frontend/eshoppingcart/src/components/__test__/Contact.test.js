import React from "react";
import ReactDOM from "react-dom";
import About from "../Contact";

import { render } from "@testing-library/react";
// import "jest-dom/extend-expect";
import "@testing-library/jest-dom";

import renderer from "react-test-renderer";

it("matches snapshot", () => {
  const tree = renderer.create(<About></About>).toJSON();
  expect(tree).toMatchSnapshot();
});
