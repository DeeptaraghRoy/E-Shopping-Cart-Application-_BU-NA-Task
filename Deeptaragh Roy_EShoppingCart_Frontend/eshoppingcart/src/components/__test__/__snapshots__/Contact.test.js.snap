// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="mb-5 container"
>
  <div
    className="row"
  >
    <div
      className="text-center py-4 my-4 col"
    >
      <h1>
        Have Some Questions?
      </h1>
      <hr />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="d-flex justify-content-center col"
    >
      <img
        alt="Contact Us"
        height="400px"
        src="/assets/images/contact.jpg"
        width="550px"
      />
    </div>
    <div
      className="col"
    >
      <form
        className=""
      >
        <div
          className="mb-3"
        >
          <label
            className="form-label"
            htmlFor="exampleForm"
          >
            Full Name
          </label>
          <input
            className="form-control"
            id="exampleForm"
            placeholder="FirstName LastName"
            type="text"
          />
        </div>
        <div
          className="mb-3"
        >
          <label
            className="form-label"
            htmlFor="exampleForm"
          >
            Email Address
          </label>
          <input
            className="form-control"
            id="exampleForm"
            placeholder="name@example.com"
            type="email"
          />
        </div>
        <div
          className="mb-3"
        >
          <label
            className="form-label"
            htmlFor="exampleForm.ControlTextarea1"
          >
            Query
          </label>
          <textarea
            className="form-control"
            id="exampleForm.ControlTextarea1"
            rows={3}
          />
        </div>
        <button
          className="btn btn-outline-primary ms-2"
          disabled={false}
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </div>
</div>
`;
